.github/conda/meta.yaml

```diff
@@ -16,7 +16,6 @@ requirements:
     - pip
     - numpy >=1.17
     - dataclasses
-    - importlib_metadata
```

Comment:
I am not very sure what this file is for. It's under .github/conda ..(?)

Reply:
I think it automates the releases on conda, but basically it doesn't need dataclasses anymore (was for Python 3.7) and won't need `importlib_metadata` after python 3.8